src/types.jl

```diff
@@ -462,15 +445,15 @@ interface described there.
     close atoms and save on computation.
 - `loggers::L=()`: the loggers that record properties of interest during a
     simulation.
-- `k::K=Unitful.k`: the Boltzmann constant, which may be modified in some
-    simulations.
+- `k::K=Unitful.k or Unitful.k*Unitful.Na`: the Boltzmann constant, which may be modified in some
+    simulations. k is chosen based on `energy_units` passed.
 - `force_units::F=u"kJ * mol^-1 * nm^-1"`: the units of force of the system.
     Should be set to `NoUnits` if units are not being used.
 - `energy_units::E=u"kJ * mol^-1"`: the units of energy of the system. Should
     be set to `NoUnits` if units are not being used.
 """
 mutable struct System{D, G, T, A, C, B, V, AD, TO, PI, SI, GI, CN, NF,
-                      L, K, F, E, M} <: AbstractSystem{D}
```

Comment:
Leave this order as is to avoid changing the API where possible (also below).

Reply:
Does that also include the order of kwargs or just the order of types in the constructors?

Reply:
Like I can revert it back to L, K, F, E, M but I think `k` has to be the last kwarg (or at least after `energy_units`)

Reply:
Just the type order.